backend/migrate.sh

```
@@ -11,16 +11,41 @@ echo "Creating databases if not exist..."
 export PGPASSWORD="$POSTGRES_PASSWORD"
 
 psql -h "$POSTGRES_HOST" -U "$POSTGRES_USER" -tc "SELECT 1 FROM pg_database WHERE datname='flowable'" | grep -q 1 || \
-psql -h "$POSTGRES_HOST" -U "$POSTGRES_USER" -c "CREATE DATABASE flowable"
+psql -h "$POSTGRES_HOST" -U "$POSTGR#!/bin/sh
```

Shellcheck (0.11.0)

[warning] Did you forget to close this double quoted string?

(SC1078)